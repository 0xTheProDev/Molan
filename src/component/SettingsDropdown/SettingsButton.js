// @flow
import React from 'react';
import { Button, Icon } from 'semantic-ui-react';

export const SettingsButton = (
  <Button floated="right" title="Settings" basic>
    <Icon name="settings" />
  </Button>
);
