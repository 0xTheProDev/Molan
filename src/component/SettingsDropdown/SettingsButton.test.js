import React from 'react';

import { SettingsButton } from './SettingsButton';

describe('Settings Button', () => {

  it('should render', () => {
    const wrapper = shallow(SettingsButton);
    expect(wrapper).toMatchSnapshot();
  });
});
