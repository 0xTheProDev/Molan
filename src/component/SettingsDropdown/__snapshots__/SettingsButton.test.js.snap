// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Button should render 1`] = `
<Ref
  innerRef={
    Object {
      "current": null,
    }
  }
>
  <button
    className="ui basic right floated button"
    onClick={[Function]}
    title="Settings"
  >
    <Icon
      as="i"
      name="settings"
    />
  </button>
</Ref>
`;
